### ISSUE TITLE
[Bug]: Filtering by AssetStatus not working on GetAssetsRequest()

### ISSUE THREAD

**Issue by iguy0** [alpacahq/alpaca-py#249]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When tryng to filter by AssetStatus i get: NameError:  name 'AssetStatus' is not defined. Did you mean: 'AssetClass'?
trading_client = TradingClient(ALPACA_API_KEY, ALPACA_API_SECRET)
search_params = GetAssetsRequest(status=AssetStatus.ACTIVE, asset_class=AssetClass.US_EQUITY)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'AssetStatus' is not defined. Did you mean: 'AssetClass'?

The documentation from here: https://alpaca.markets/docs/python-sdk/trading.html
states it should be possible.

### Expected Behavior

Return correct search_params variable

### SDK Version I encountered this issue in

latest install from pip

### Steps To Reproduce

```markdown
import os
from dotenv import load_dotenv
from alpaca.trading.client import TradingClient
from alpaca.trading.requests import GetAssetsRequest
from alpaca.trading.enums import AssetClass

load_dotenv()
ALPACA_API_KEY = os.getenv('ALPACA_API_KEY')
ALPACA_API_SECRET = os.getenv('ALPACA_API_SECRET')

trading_client = TradingClient(ALPACA_API_KEY, ALPACA_API_SECRET)
search_params = GetAssetsRequest(status=AssetStatus.ACTIVE, asset_class=AssetClass.US_EQUITY)
```

### Filled out the Steps to Reproduce section?

- [X] I have entered valid steps to reproduce my issue or have attached a minimally reproducible case in code that shows my issue happening; and understand that without this my issue will be flagged as invalid and closed after 30 days.

### Anything else?

Seems like AssetStatus is not implemented.

**Comment by alessiocastrica** [comment]
@iguy0 you didn't import AssetStatus from the enums, that's what is causing this issue.

**Comment by iguy0** [comment]
You're correct, It worked. I wish the documentation would be a bit more descriptive of this.  Thanks for the hint!
